src/main/java/com/bunq/sdk/context/ApiContext.java

```diff
@@ -124,14 +123,20 @@ public static ApiContext restore(String fileName) {
     try {
       File file = new File(fileName);
       String json = FileUtils.readFileToString(file, ENCODING_BUNQ_CONF);
-      JsonObject jsonObject = gson.fromJson(json, JsonObject.class);
 
-      return gson.fromJson(jsonObject, ApiContext.class);
+      return fromJson(json);
     } catch (IOException exception) {
       throw new BunqException(ERROR_COULD_NOT_RESTORE_API_CONTEXT, exception);
     }
   }
 
+  /**
+   * Restores a context from a given JSON string.
+   */
+  public static ApiContext fromJson(String json) {
+    return gson.fromJson(json, ApiContext.class);
+  }
+
   private void initialize(String deviceDescription, List<String> permittedIps) {
     /* The calls below are order-sensitive: to initialize a Device Registration, we need an
      * Installation, and to initialize a Session we need a Device Registration. */
```

Comment:
@OGKevin The format you've specified is Doc Block comment which can be extracted to a generated documentation. This comment is opposite. It is just a normal multi-line comment which is only needed to explain the internals of the method, not the method itself.

So, Java has:
```
/**
 * Doc Block
 */
```

and

```
/*
 * Just a multi-line comment.
 */